.claude/agents/tester.md

```
@@ -1,171 +1,78 @@
+
+### 4. Run Tests
+```bash
+go test ./internal/markdown/... -v -run {relevant_test}
+go build ./...
 ```
 
-**For Forms:**
+## Output
+
+Success:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)